features/fork.feature

```diff
@@ -152,6 +152,33 @@ Scenario: Related fork already exists
       """
     And the url for "mislav" should be "git@github.com:mislav/unrelated.git"
 
+ Scenario: Related fork and related remote already exist
+     Given the "mislav" remote has url "git@github.com:mislav/dotfiles.git"
+     Given the GitHub API server:
+     """
+       get('/repos/mislav/dotfiles') {
+             json :parent => { :html_url => 'https://github.com/EvilChelu/Dotfiles' }
+           }
+     """
+     When I run `hub fork`
+     Then the exit status should be 0
+     And the url for "mislav" should be "git@github.com:mislav/dotfiles.git"
+
+  Scenario: Related fork and related remote, but with differing protocol, already exist
+      Given the "mislav" remote has url "https://github.com/mislav/dotfiles.git"
+      Given the GitHub API server:
+      """
+        get('/repos/mislav/dotfiles') {
+              json :parent => { :html_url => 'https://github.com/EvilChelu/Dotfiles' }
+            }
+      """
+      When I run `hub fork`
+      Then the exit status should be 128
```

Comment:
As I've stated in above comment, I think this scenario should exit with success status and keep the `mislav` remote intact. The difference in protocol shouldn't matter (the orignal protocol of the existing `mislav` remote should "win").